### PR TITLE
Sonicwall FW shipping 

### PR DESCRIPTION
**Pull request by shalper**
# What changed

----

## Pages to review

- https://deploy-preview-544--logz-docs.netlify.app/shipping/log-sources/sonicwall.html

### CODE REVIEW

File: _source/logzio_collections/_log-sources/sonicwall.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+* [Filebeat 7](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-installation.html) or [Filebeat 6](https://www.elastic.co/guide/en/beats/filebeat/6.7/filebeat-installation.html)
+* root access
+
+##### Configure Cisco ASA Server logging
+
+Configure your Cisco ASA firewall to send logs to your Filebeat server.
+Make sure you meet this configuration:
+
+* Log format: syslog
+* Send over: TCP
```
Comment (FatnLazy) [inline _source/logzio_collections/_log-sources/sonicwall.md:31]: Log format must be UDP

File: _source/logzio_collections/_log-sources/sonicwall.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+* IP address: Filebeat server IP address
+* Port 6514
```
Comment (FatnLazy) [inline _source/logzio_collections/_log-sources/sonicwall.md:33]: Port 514

File: _source/logzio_collections/_log-sources/sonicwall.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+* [Filebeat 7](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-installation.html)
+* root access
+
+##### Configure SonicWall Server logging
+
+Configure your SonicWall firewall to send logs to your Filebeat server.
+Make sure you meet this configuration:
+
+* Log format: syslog
+* Send over: UDP
```
Comment (shalper) [inline _source/logzio_collections/_log-sources/sonicwall.md:32]:
```suggestion
```

File: _source/logzio_collections/_log-sources/sonicwall.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+##### Configure SonicWall Server logging
+
+Configure your SonicWall firewall to send logs to your Filebeat server.
+Make sure you meet this configuration:
+
+* Log format: syslog
+* IP address: Filebeat server IP address
+* Port 514
+
```
Comment (shalper) [inline _source/logzio_collections/_log-sources/sonicwall.md:34]:
```suggestion

* (SonicWall firewall sends logs over UDP by default.) 
```

File: _source/logzio_collections/_log-sources/sonicwall.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
+---
+title: Ship SonicWall logs
+logo:
+  logofile: SonicWall-Logo.svg
```
Comment (shalper) [inline _source/logzio_collections/_log-sources/sonicwall.md:4]:
```suggestion
  logofile: sonicwall-small.svg
```

File: _source/logzio_collections/_log-sources/sonicwall.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
+---
+title: Ship SonicWall logs
+logo:
+  logofile: sonicwall-small.svg
+  orientation: vertical
```
Comment (shalper) [inline _source/logzio_collections/_log-sources/sonicwall.md:5]:
```suggestion
  logofile: SonicWall-Logo.svg
  orientation: horizontal
```

File: _source/logzio_collections/_log-sources/sonicwall.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+Configure your SonicWall firewall to send logs to your Filebeat server.
+Make sure you meet this configuration:
+
+* Log format: syslog
+* IP address: Filebeat server IP address
+* Port 514
+
+* (SonicWall firewall sends logs over UDP by default.) 
+See [SonicWall docs](https://www.sonicwall.com/support/technical-documentation/?language=English) for more information
+on configuring your SonicWall firewall.
```
Comment (shalper) [inline _source/logzio_collections/_log-sources/sonicwall.md:37]:
```suggestion
* Log format: syslog
* IP address: Filebeat server IP address
* Port 514

SonicWall firewall sends logs over UDP by default. See [SonicWall docs](https://www.sonicwall.com/support/technical-documentation/?language=English) for more information
on configuring your SonicWall firewall.
```

**Pull request merged**